src/components/Auth/UserFormDialog/UserForm.tsx

```diff
@@ -65,23 +65,23 @@ function convertFromFormUser(formUser: AuthFormUser): AddAuthUserPayload {
   return {
     ...formUser,
```

Comment:
I wonder if we should remove extra fields like `mfaPhoneInfo` from the payload as well? Those fields are only used in the form and won't make sense when passed into the API.

Reply:
I believe they're gone already, since this function returns an `AddAuthUserPayload`: https://github.com/firebase/firebase-tools-ui/blob/211bd5bbcf85ef2f8c47a6e4401ae087aea16f6c/src/components/Auth/types.ts#L41-L51

Which contains these:

https://github.com/firebase/firebase-tools-ui/blob/211bd5bbcf85ef2f8c47a6e4401ae087aea16f6c/src/components/Auth/types.ts#L29-L35

Or am I missing it somewhere else?

Reply:
`formUser` is of type `AuthFormUser`, which contains three extra fields: `mfaPhoneInfo`, `mfaEnabled` and `emailVerified`. The code here copies over them (by splating `formUser`) and then overrides `emailVerified`, but I believe `mfaEnabled` and `mfaPhoneInfo` are still included in the resulting object.

Note that `mfaPhoneInfo`, unlike `mfaInfo`, isn't part of the API contract. (Also note that TypeScript won't catch these extra fields since this object is technically compatible with `AddAuthUserPayload` since it has a super set of fields. TS only complains about object literals having extra fields when splats are not used)